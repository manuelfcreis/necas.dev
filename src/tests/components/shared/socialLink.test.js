import React from "react";
import SocialLink from "../../../components/shared/socialLink";
import { shallow } from "enzyme";

it("renders correctly", () => {
  const element = shallow(
    <SocialLink link="https://test.com" image="test.png" alt="test" />
  );
  expect(element).toMatchSnapshot();
});
