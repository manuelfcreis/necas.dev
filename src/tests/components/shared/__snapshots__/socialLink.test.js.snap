// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<li>
  <a
    className="icon"
    href="https://test.com"
  >
    <img
      alt="test"
      src="test.png"
    />
  </a>
</li>
`;
